source/_integrations/knx.markdown

```diff
@@ -108,6 +108,33 @@ state_updater:
   type: boolean
 {% endconfiguration %}
 
+### Group addresses
+
+Group addresses are configured as strings or integers in the format "1/2/3" for 3-level GA-structure, "1/2" for 2-level GA-structure or "1" for free GA-structure.
+
+The HA KNX integration uses configured `state_address` or `*_state_address` to update the state of a function. These addresses are read by GroupValueRead requests on startup and when there was no incoming telegram for one hour (default `sync_state`).
+
+It is possible to configure passive / listening group addresses for all functions of every KNX platform (except `expose` and `notify`). This allows to have multiple group addresses to update the state of its function (e.g. brightness of a light). When group addresses are configured as list of strings, the first item is the active sending or state-reading address and the rest is registered as passive addresses. This schema behaves like in ETS configuration where the first is the "sending" address and others are just for updating the communication object.
+
+If your KNX device provides active state communication objects it is advised to use `*_state_address` instead of passive addresses as it reduces configuration complexity and avoids wrong states (e.g. when channels are logically locked).
```

Comment:
```suggestion
If your KNX device provides active state communication objects it is advised to use `*_state_address` instead of passive addresses as it reduces configuration complexity and avoids wrong states (e.g., when channels are logically locked).
```